files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md

```diff
@@ -172,7 +172,7 @@ p
 
 ![](locallibary_express_author_delete_withbooks.png)
 
-> **备注：** 其他删除对象的页面，可以用相同的方式实现。我们把它留下，作为挑战。
+> **注意：** 其他删除对象的页面，可以用相同的方式实现。我们把它留下，作为挑战。
```

Comment:
idto

Reply:
maybe needs to add a notes for these kinds of translation in the guidelines?

Reply:
```suggestion
> **备注：** 其他删除对象的页面，可以用相同的方式实现。我们把它留下，作为挑战。
```